src/pages/speakers/speakers-tips/index.astro

```diff
@@ -7,27 +7,27 @@ import {Image} from "astro:assets"
 <Layout title="JavaZone 2024 - Speakers - Submission tips">
```

Comment:
I denne siden er også marginer på PC-skjermer for små, siden vises i full bredde. Kan det gjøres noe med det?

Reply:
fikset:)